src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx

```diff
@@ -30,7 +38,53 @@ export const ModalAddMembers = ({
 }: ModalAddMembersProps) => {
   const { t } = useTranslation();
   const [selectedMembers, setSelectedMembers] = useState<OptionSelect>([]);
-  const [, setSelectedRole] = useState<Role>(Role.MEMBER);
+  const [selectedRole, setSelectedRole] = useState<Role>(Role.MEMBER);
+  const { toast } = useToastProvider();
+  const { mutateAsync: createInvitation } = useCreateInvitation();
+
+  const handleValidate = async () => {
+    const promises = await Promise.allSettled<Invitation>(
+      selectedMembers.map((selectedMember) => {
+        return new Promise<Invitation>((resolve, reject) => {
+          createInvitation({
+            email: selectedMember.value.email,
+            role: selectedRole,
+            teamId: team.id,
+          })
+            .then((data) => {
+              resolve(data);
+            })
+            .catch((error) => {
+              reject(error);
+            });
+        });
+      }),
+    );
+
+    onClose();
+    promises.forEach((promise) => {
+      switch (promise.status) {
+        case 'rejected':
```

Comment:
I don't know if this important, but Invitation error are not translated.

Reply:
https://github.com/numerique-gouv/people/pull/131/commits/a31c16eb2de3515b3257fa6a5c776757baa515b5

Reply:
https://github.com/numerique-gouv/people/pull/131/commits/a521646dab36d53a2e473e51cfedb74258131416#diff-1c926ca4662a2efb0ebca0a591e1243d1322baaab47ee82337e65b73a66974eeR60-R64